From 37c6eb57cb59b8af5b9c6233f6db85129e4727d8 Mon Sep 17 00:00:00 2001
From: Peng Ma <peng.ma@nxp.com>
Date: Thu, 13 Dec 2018 15:13:44 +0800
Subject: [PATCH 405/706] arm64: dts: ls1028a: enable little endian for edma

the edma of ls1028a soc is little endian, so we should change the big
endian to little endian

Signed-off-by: Peng Ma <peng.ma@nxp.com>
(cherry picked from commit 66cfc95fa538d8be07fcf415e98d01840b003f01)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index af8db8fe5339..83db88cdaba0 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -739,7 +739,6 @@
 				     <0 56 0x4>;
 			interrupt-names = "edma-tx", "edma-err";
 			dma-channels = <32>;
-			big-endian;
 			clock-names = "dmamux0", "dmamux1";
 			clocks = <&clockgen 4 1>,
 				 <&clockgen 4 1>;
-- 
2.17.1

